packages/react-focus/README.md

```diff
@@ -1,11 +1,15 @@
 # @fluentui/react-focus
 
-**React Focus components for [Fluent UI](https://fluent.microsoft.com/)**
+**Components that deal with focus management for [Fluent UI](https://fluent.microsoft.com/)**
```

Comment:
Wording suggestion plus name and link fix--we should link to our site, not the fluent design system site
```suggestion
**Focus management components for [Fluent UI React](https://dev.microsoft.com/fabric)**
```